Jenkinsfile.integration

```diff
@@ -23,7 +23,6 @@ pipeline {
     parameters {
         /* first value in the list is the default */
         choice(choices: ['airship', 'osh'], description: 'Which deployment mechanism?', name: 'deployment')
-        booleanParam(name: 'run_tempest', defaultValue: false, description: 'Run tempest tests?')
```

Comment:
I'd prefer this would remain configurable, but the default value to True. Maybe in the future we want to skip these tests for some scenarios.

Reply:
Card AC says that it should be removed so....I also think it would be good to have it just in case, for manual tests that do not necessarily need a full tempest run.

Reply:
If it makes sense to leave it, then lets do so and change the card. I'm happy either way.

Reply:
I rather not have to bring an extra option around...plus I would need to change both PRs :P

Reply:
I guess it's ok, we can always bring that back if needed.

Reply:
Let's solemnly vow we'll never bring that back, and always run tempest!